test-cypress/integration/explore/visual.graph.spec.js

```diff
@@ -382,393 +228,352 @@ describe('Visual graph screen validation', function () {
+    }
+
+    function searchForResource(resource) {
+        typeInSearchField(resource);
+        // Verify redirection to existing visual graph
+        cy.get('.nodes-container').should('be.visible');
+    }
+
+    function getTargetNodeElement() {
+        return cy.get(`[id="http://www.w3.org/TR/2003/PR-owl-guide-20031209/wine#${VALID_RESOURCE}"] circle`);
+    }
+
+    function getTargetNode() {
+        // timeout is needed because mouseover event will result in pop-up of menu icons only if nodes are not moving
+        return getTargetNodeElement().should('be.visible').wait(5000);
+    }
+
+    function getNodes() {
+        return cy.get('.node-wrapper');
+    }
+
+    function getPredicates() {
+        return cy.get('.predicate');
+    }
+
+    function getNodeInfoPanel() {
+        return cy.get('.rdf-info-side-panel .tab-content');
+    }
+
+    // Visual graph settings form field access
+
+    function openPredicatesTab() {
+        cy.get('.predicates-tab').click();
+    }
+
+    function showPreferredTypes(enable) {
+        let command = enable ? 'check' : 'uncheck';
+        getShowPreferredTypesOnlyCheckbox()[command]();
+    }
+
+    function enableInferredStatements(enable) {
+        openVisualGraphSettings();
+        let command = enable ? 'check' : 'uncheck';
+        getIncludeInferredStatementsCheckbox()[command]();
+        saveSettings();
+    }
+
+    function getLinksNumberField() {
+        return cy.get('.input-number');
+    }
+
+    function getSaveSettingsButton() {
+        return cy.get('.save-settings-btn');
+    }
+
+    function saveSettings() {
+        getSaveSettingsButton().click();
+    }
+
+    function getResetSettingsButton() {
+        return cy.get('.reset-settings');
+    }
+
+    function resetSettings() {
+        getResetSettingsButton().click();
+    }
+
+    function getSameAsCheckbox() {
+        return cy.get('#sameAsCheck');
+    }
+
+    function getIncludeInferredStatementsCheckbox() {
+        return cy.get('.include-inferred-statements');
+    }
+
+    function getShowPredicateLabelsCheckbox() {
+        return cy.get('.show-predicate-labels');
+    }
+
+    function getPreferredTypesField() {
+        return cy.get('.preferred-types input');
+    }
+
+    function getShowPreferredTypesOnlyCheckbox() {
+        return cy.get('.show-preferred-types-only');
+    }
+
+    function getIgnoredTypesField() {
+        return cy.get('.ignored-types input');
+    }
+
+    function getPreferredPredicatesField() {
+        return cy.get('.preferred-predicates input');
+    }
+
+    function getShowPreferredPredicatesOnlyCheckbox() {
+        return cy.get('.show-preferred-predicates-only');
+    }
+
+    function getIgnoredPredicatesField() {
+        return cy.get('.ignored-predicates input');
+    }
+
+    // Node actions
+
+    function collapseGraph() {
+        cy.get('.menu-events .collapse-icon circle').should('be.visible').click();
```

Comment:
`should('be.visible')` is not needed if you are not forcing the click

Reply:
Yeah, my bad.